packages/next/package.json

```diff
@@ -73,7 +73,8 @@
     "caniuse-lite": "^1.0.30001283",
     "postcss": "8.4.5",
     "styled-jsx": "5.0.0",
-    "use-subscription": "1.5.1"
+    "use-subscription": "1.5.1",
+    "web-streams-polyfill": "3.0.3"
```

Comment:
this is already in `compiled` but it has no type definition there. How can I make sure that TypeScript definitions are being used there as well? Should I copy/reference them manually in the `package.json`?